src/fswAlgorithms/attGuidance/hillPointCpp/hillPointCpp.cpp

```diff
@@ -22,53 +22,29 @@
  */
 
 
-#include "fswAlgorithms/attGuidance/hillPoint/hillPoint.h"
-#include <string.h>
-#include "fswAlgorithms/fswUtilities/fswDefinitions.h"
-#include "architecture/utilities/macroDefinitions.h"
+#include "hillPointCpp.h"
 
 /* Support files.  Be sure to use the absolute path relative to Basilisk directory. */
 #include "architecture/utilities/linearAlgebra.h"
 #include "architecture/utilities/rigidBodyKinematics.h"
 
 
-
-/*! self init method
- @return void
- @param configData The configuration data associated with RW null space model
- @param moduleID The ID associated with the configData
- */
-void SelfInit_hillPoint(hillPointConfig *configData, int64_t moduleID)
-{
-    AttRefMsg_C_init(&configData->attRefOutMsg);
-}
-
-
-/*! This method performs the module reset capability.  This module has no actions.
- @return void
- @param configData The configuration data associated with this module
- @param callTime The clock time at which the function was called (nanoseconds)
- @param moduleID The ID associated with the configData
- */
-void Reset_hillPoint(hillPointConfig *configData, uint64_t callTime, int64_t moduleID)
-{
+/*! This method performs the module reset capability.  This module has no actions. */
+void HillPointCpp::Reset(uint64_t CurrentSimNanos){
     // check if the required input message is included
-    if (!NavTransMsg_C_isLinked(&configData->transNavInMsg)) {
-        _bskLog(configData->bskLogger, BSK_ERROR, "Error: hillPoint.transNavInMsg wasn't connected.");
+    if (!NavTransMsg_C_isLinked(&this->transNavInMsg)) {
+        _bskLog(this->bskLogger, BSK_ERROR, "Error: hillPoint.transNavInMsg wasn't connected.");
```

Comment:
`Error:` does not need to be in the error message, as this will already be output. 

Also this function call should change to the C++ version E.g.
`this->bskLogger->bskLog(BSK_ERROR, "hillPoint.transNavInMsg wasn't connected.");`
(Pat requested this change from me on a previous PR as well)

Reply:
Done